### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by lijian-1994** [meijieru/crnn.pytorch#180]
Ubuntu 17.04  
no gpu
python 3.6
cs316@xxxx:~/lijian/crnn.pytorch-master$ python train.py -- trainRoot ./data/train --valRoot ./data/val  --random_sample  --workers 1
Namespace(adadelta=False, adam=False, alphabet='0123456789abcdefghijklmnopqrstuv                             wxyz/', batchSize=64, beta1=0.5, cuda=False, displayInterval=500, expr_dir='expr                             ', imgH=32, imgW=100, keep_ratio=False, lr=0.01, manualSeed=1234, n_test_disp=10                             , nepoch=25, ngpu=0, nh=256, pretrained='', random_sample=True, saveInterval=500                             , trainRoot='./data/train', valInterval=500, valRoot='./data/val', workers=1)
CRNN(
  (cnn): Sequential(
    (conv0): Conv2d(1, 64, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu0): ReLU(inplace)
    (pooling0): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_m                             ode=False)
    (conv1): Conv2d(64, 128, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu1): ReLU(inplace)
    (pooling1): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_m                             ode=False)
    (conv2): Conv2d(128, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (batchnorm2): BatchNorm2d(256, eps=1e-05, momentum=0.1, affine=True, track_r                             unning_stats=True)
    (relu2): ReLU(inplace)
    (conv3): Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu3): ReLU(inplace)
    (pooling2): MaxPool2d(kernel_size=(2, 2), stride=(2, 1), padding=(0, 1), dil                             ation=1, ceil_mode=False)
    (conv4): Conv2d(256, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (batchnorm4): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True, track_r                             unning_stats=True)
    (relu4): ReLU(inplace)
    (conv5): Conv2d(512, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (relu5): ReLU(inplace)
    (pooling3): MaxPool2d(kernel_size=(2, 2), stride=(2, 1), padding=(0, 1), dil                             ation=1, ceil_mode=False)
    (conv6): Conv2d(512, 512, kernel_size=(2, 2), stride=(1, 1))
    (batchnorm6): BatchNorm2d(512, eps=1e-05, momentum=0.1, affine=True, track_r                             unning_stats=True)
    (relu6): ReLU(inplace)
  )
  (rnn): Sequential(
    (0): BidirectionalLSTM(
      (rnn): LSTM(512, 256, bidirectional=True)
      (embedding): Linear(in_features=512, out_features=256, bias=True)
    )
    (1): BidirectionalLSTM(
      (rnn): LSTM(256, 256, bidirectional=True)
      (embedding): Linear(in_features=512, out_features=38, bias=True)
    )
  )
)
Segmentation fault (core dumped)

when i run,it occur error 'Segmentation fault (core dumped)' and don't have other error message.
who can help me?

**Comment by Xassassinator** [comment]
@lijian-1994 I have met the same issue, did you solved this problem? could you tell me about the details, thank you 
!

**Comment by gzhcv** [comment]
I got the same issue, have you sloved it?

**Comment by Pratyeka** [comment]
I had the same problem，what is the cause of this problem and what's the solution

**Comment by zhengjiawen** [comment]
Hi, how did you slove the problem? Can you please share? Thaks.

**Comment by Marchbruno09** [comment]
Hi, have u solved this problem?

**Comment by keevinzha** [comment]
I got the same problem, have someone solved it now?

**Comment by longbowking** [comment]
What is going wrong,  why "Segmentation fault"?
i cannot believe a project with 1.1k stars has such issues, until they occur to me!
It is quite frustrating.

**Comment by yunchangxiaoguan** [comment]
wtf！too much problem in this code

**Comment by apple2333cream** [comment]
I got the same issue, have you sloved it?
